kolibri/plugins/learn/assets/src/views/classes/LessonPlaylistPage.vue

```diff
@@ -52,20 +52,22 @@
   import ContentIcon from 'kolibri.coreVue.components.ContentIcon';
   import { getContentNodeThumbnail } from 'kolibri.utils.contentNode';
   import ContentCard from '../ContentCard';
+  import { coachStringsMixin } from '../../../../../coach/assets/src/views/common/commonCoachStrings';
   import { lessonResourceViewerLink } from './classPageLinks';
 
   export default {
     name: 'LessonPlaylistPage',
     metaInfo() {
       return {
-        title: this.$tr('documentTitle'),
```

Comment:
@indirectlylit Did we have guidelines for combinations of user-generated and our strings?

Reply:
When strings are used in a phrase, they [should be wrapped](https://design-system.learningequality.org/writing/#notes) in 'single quotes'.

```
Would you like to delete the lesson '{lesson_name}'?
```

If a user-generated string , I think colons are more typical than dashes:

```
Lesson: {lesson_name}
```

If this is being used to show a page hierarchy, we do concatenate with dashes - for example:

https://github.com/learningequality/kolibri/blob/60cd4eb6338bef0151fbc7739ee0641049f07d76/kolibri/plugins/coach/assets/src/views/common.js#L42-L80

However it's not clear to me that this is quite correct, and relates to the [thread started in Slack](https://learningequality.slack.com/archives/C59J8FEPN/p1631207348030200) where @rtibbles suggested looking at [`Intl.ListFormat`](https://formatjs.io/docs/polyfills/intl-listformat/)

Reply:
Thank you! My idea was to use this string for the page title, so I was thinking of some hierarchy... However, since we decided with @rtibbles to hold off on working on #8417 for 0.15, we can't have the programmatic concatenation, yet.

So let's settle on the simplest solution for the moment:

`title: this.currentLesson.title,`